res/TemplateBatchFiles/MEDS.bf

```diff
@@ -1,479 +0,0 @@
-RequireVersion ("2.00.20101201");
```

Comment:
Just want to make sure we do in fact want to delete the MEDS batch file?

Reply:
I think so. The goal is to have it as a stand-alone analysis in the veg/hyphy-analyses repo. 
We could hold off on removing until that is running as the current bf does run with 2.4.

Reply:
Ahh... gotcha. I think it makes sense to remove it now if the current bf doesn't run. Thanks!